src/status_im/chat/views/message/message.cljs

```diff
@@ -164,9 +164,11 @@
 
 (defn text-message
   [{:keys [content] :as message}]
-  [message-view message
-   (let [parsed-text (cached-parse-text content :browse-link-from-message)]
-     [react/text {:style (style/text-message message)} parsed-text])])
+  (let [parsed-text (cached-parse-text content :browse-link-from-message),
+        emoji-only @(re-frame/subscribe [:message-contains-only-emoji content])]
+    [message-view (merge message {
+                   :no-bubble emoji-only})
```

Comment:
Why `merge` but not `assoc`?
Also, strange looking `{` left at the end of the line (formatting)